test/controllers/api_users/permissions_controller_test.rb

```diff
@@ -194,24 +194,42 @@ class ApiUsers::PermissionsControllerTest < ActionController::TestCase
       assert_same_elements [other_permission, not_from_ui_permission], api_user.supported_permissions
     end
 
-    should "prevent permissions being added for other apps" do
```

Comment:
Typo in the commit message (copied over from my own typo!): `included` -> `includes`